static/homepage.json

```diff
@@ -42,10 +119,319 @@
+            "n_threads": 1,
+            "dev_name": "lightning-dev",
+            "contributors": []
+        },
+        {
+            "id": "004004",
+            "title": "HTLC Endorsement for Jamming Mitigation",
+            "link": "https://lists.linuxfoundation.org/pipermail/lightning-dev/2023-July/004004.html",
+            "authors": [
+                "Antoine Riard"
+            ],
+            "published_at": "2023-07-05T20:11:52+00:00",
+            "summary": "1. Participants discuss potential damage from routing and on-chain fees, propose different thresholds for routing fees.2. Reputation-based and monetary-based mitigation strategies explored, with diverging opinions on reliance solely on reputation.3. Challenges of mitigating DDoS attacks in the Lightning Network context highlighted, need for rigorous analysis emphasized.",
+            "n_threads": 18,
+            "dev_name": "lightning-dev",
+            "contributors": [
+                "Carla Kirk-Cohen",
+                "Christian Decker",
+                "Clara Shikhelman",
+                "Michael Folkson",
+                "Vincenzo Palazzo"
+            ]
+        },
+        {
+            "id": "004003",
+            "title": "Computing Blinding Factors in a PTLC and Trampoline World",
+            "link": "https://lists.linuxfoundation.org/pipermail/lightning-dev/2023-July/004003.html",
+            "authors": [
+                "ZmnSCPxj"
+            ],
+            "published_at": "2023-07-04T20:55:34+00:00",
+            "summary": "1. Bastien and ZmnSCPxj discuss a proposal for a schnorr multi-hop locks mechanism in their conversation.2. ZmnSCPxj argues that their proposal is a restatement of the same scheme specified in a previous proposal.3. ZmnSCPxj's proposal simplifies the process by eliminating the need to send left and right locks when using a trampoline node.",
+            "n_threads": 3,
+            "dev_name": "lightning-dev",
+            "contributors": [
+                "Bastien TEINTURIER"
+            ]
+        }
+    ],
+    "active_posts": [
+        {
+            "id": "021651",
+            "title": "Bitcoin Core maintainers and communication on merge decisions",
+            "link": "https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2023-May/021651.html",
+            "authors": [
+                "Steve Lee"
+            ],
+            "published_at": "2023-05-10T02:44:59+00:00",
+            "summary": "1. Discussion on Bitcoin development mailing list raises concerns about decision-making power of current maintainers.2. Bitcoin Core should function as a volunteer project, not controlled by a select few.3. Confusion between Bitcoin Core and bitcoin protocol emphasizes the need for clear governance structures.",
```

Comment:
please include a new line between the unordered list items so that it's easy to parse on the front end